src/convertToObject.js

```diff
@@ -5,8 +5,18 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(str) {
+  const styleObject = {};
+
+  str.split(';').forEach((el) => {
+    const [property, value] = el.split(':');
+
+    if (property && value) {
+      styleObject[property.trim()] = value.trim();
+    }
+  });
```

Comment:
[CODE PRACTICE] - Instead of using the `forEach` method to modify the `styleObject`, consider using `reduce` which is more suited for transforming an array into a single output value like an object. This aligns with best practices and the checklist item 5 which suggests avoiding loops and using iteration methods instead.